polynote-kernel/src/main/scala/polynote/kernel/interpreter/package.scala

```diff
@@ -38,4 +42,37 @@ package object interpreter {
     def access: URIO[InterpreterState, Service] = ZIO.service[Service]
   }
 
+  def posToLineAndColumn(code: String, pos: Int): (Int, Int) = {
```

Comment:
Worth a unit test? This sort of logic can get finicky

Reply:
Good call – added a test and found a bug!